src/Stryker.Core/Stryker.Core/Mutants/MutantPlacer.cs

```diff
@@ -7,6 +7,22 @@ namespace Stryker.Core.Mutants
 {
     public static class MutantPlacer
     {
+        private const string helper = @"
+namespace Stryker
```

Comment:
Maybe it would be nice to actually include this as a C# file. That way we can test this code as well and have syntax checking. We can just read the file and parse it here.

Reply:
Done. Note that I had to change the naming of the helper class to prevent ambiguity with System.Environment.